### PR TITLE
feat: v4 header stats [2/n]

### PR DESCRIPTION
**Pull request by johnnyd-eth**
Screenshot 2024-07-24 at 1 28 47 pm

### CODE REVIEW

File: src/packages/v4/views/V4ProjectDashboard/ProjectHeaderStats.tsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+  return (
+    <div className="flex min-w-0 gap-12 md:shrink-0">
+      <a role="button" onClick={openActivityTab}>
+        <ProjectHeaderStat label={t`Payments`} stat={payments ?? 0} />
+      </a>
+      <ProjectHeaderStat
+        label={t`Total raised`}
+        stat={
+          <ETHAmount amount={BigNumber.from(totalVolume ?? 0)} precision={2} />
```
Comment (aeolianeth) [inline src/packages/v4/views/V4ProjectDashboard/ProjectHeaderStats.tsx:29]: mmm 🤔 . Shall we extend ETHAmount to accept bigints?

Comment (aeolianeth) [reply src/packages/v4/views/V4ProjectDashboard/ProjectHeaderStats.tsx:29]: Can you just slap a TODO on this in the code? I'll do my suggestion later if you like

File: src/packages/v4/views/V4ProjectDashboard/V4ProjectTabs/V4TokensPanel/hooks/useV4TokensPanel.ts [hunk 2]
```diff
@@ -28,39 +28,38 @@ export const useV4TokensPanel = () => {
...
+    return new JBProjectToken(_userTokenBalance ?? 0n)
+  }, [_userTokenBalance])
 
   // const { totalLegacyTokenBalance, v1ClaimedBalance } =
   //   useTotalLegacyTokenBalance({ projectId })
 
-  const totalTokenSupplyFormatted = useMemo(() => {
-    return formatWad(totalTokenSupply)
-  }, [totalTokenSupply])
+  const totalTokenSupply = useMemo(() => {
```
Comment (aeolianeth) [inline src/packages/v4/views/V4ProjectDashboard/V4ProjectTabs/V4TokensPanel/hooks/useV4TokensPanel.ts:46]: fooking useMemo.... dw bout it

**Review by aeolianeth — APPROVED** [review]

File: src/packages/v4/views/V4ProjectDashboard/V4ProjectTabs/V4TokensPanel/V4TokensPanel.tsx [hunk 3]
```diff
@@ -85,7 +85,7 @@ export const V4TokensPanel = () => {
                   </div>
                 </span>
               }
-              kebabMenu={userTokenBalance > 0 ? {
+              kebabMenu={userTokenBalance.toFloat() > 0 ? {
```
[the remark this answers is not in the document]
Comment (johnnyd-eth) [reply src/packages/v4/views/V4ProjectDashboard/V4ProjectTabs/V4TokensPanel/V4TokensPanel.tsx:88]: Ah yep indeed

File: src/packages/v4/views/V4ProjectDashboard/ProjectHeaderStats.tsx (hunk 1 above)
Comment (johnnyd-eth) [reply src/packages/v4/views/V4ProjectDashboard/ProjectHeaderStats.tsx:29]: Yep we should, do you think extend or just switch it out for bigint's entirely? Added a TODO here now

**Pull request merged**